New package: SubsurfaceCore v0.7.1

JuliaRegistrator opened the pull request:
- Registering package: SubsurfaceCore
- Repository: https://github.com/ayushinav/SubsurfaceCore.jl
- Created by: @ayushinav
- Version: v0.7.1
- Commit: ddc470ac4636f6c295c3ead83dd91b2315835cfc
- Reviewed by: @ayushinav
- Reference: https://github.com/ayushinav/SubsurfaceCore.jl/commit/ddc470ac4636f6c295c3ead83dd91b2315835cfc#commitcomment-171768892
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6fc6a5cf2d2d8eb2c7184c289029d2ae19013e1b74d19d39dc39bb173ab019000ddfcbd0cbd1c4405743ff83304c81391a7f15f608b9613e2231de2e80bc8fd60ebd1e86b3d17f86c3f8e1689ec12a8eda1a066deac201c2bfb4135975512a0e6699d7ea7cbec6e8957a14ce6f115f2a3310cfe6f35912f08e57fa7dece033278b0084bd1baf8b473510a8f7a10a0ff7865acdb647c55bf9b716c245e4139908bc874ba4d5a069579b858f5833df3f5ee -->

goerz commented:
Even if the package isn't user-facing, I would still ask to add basic API Documentation, ideally in the form of the [Documenter](https://documenter.juliadocs.org/stable/)-based documentation. Without that, there isn't really a way to apply semantic versioning.

I would also note that [ProEM.jl](https://github.com/ayushinav/ProEM.jl) mentioned in the README doesn't exist. If this package is a backend mainly for [Porosity.jl](https://github.com/ayushinav/Porosity.jl), then `PorosityCore` might be a more suitable name.

ayushinav commented:
Well, I can add some API functionalities in the README section, but as you have already noticed, it's not a user interface and all of them are included in the docs of the downstream package.
`ProEM` is not public yet and it's name is not fixed either, it should hopefully come out later this month and both `ProEM` and `Porosity` constitute subsurface imaging. Until then, I'll remove its mention from README.
Thanks for the comments.`

JuliaRegistrator committed:
New package: SubsurfaceCore v0.7.1

UUID: 3a78539b-9dce-42c5-be8d-f3fa4153f7a3
Repo: https://github.com/ayushinav/SubsurfaceCore.jl.git
Tree: adfb8eb104464f18b38ba578ae401e3610e52ca5

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
